app/Http/Requests/ContactUsRequest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class ContactUsRequest extends FormRequest
+{
+    public function authorize(): bool
+    {
+        return true;
+    }
+    public function rules(): array
+    {
+        return [
+            'email' => [ 'required', 'string', 'email'],
+            'name' => [ 'required', 'min:2', 'string' ],
```

Comment:
I recommend to keep second type of data and next additional validations likes min

Reply:
Got it, I'll take that into account